core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -35,7 +35,7 @@ oppia.directive('collectionSummaryTile', [
         getThumbnailBgColor: '&thumbnailBgColor',
         isLinkedToEditorPage: '=?isLinkedToEditorPage',
         getCategory: '&category',
-        isPlaylistMode: '&playlistMode',
+        isPlaylistTile: '&playlistTile',
```

Comment:
I think you want the actual directive attr name to also be is-playlist-tile, otherwise it's as though it's naming the playlist tile that's being passed in.

Reply:
Done!